src/redux/categories/categories.js

```diff
@@ -0,0 +1,26 @@
+const CHECKED_STATUS = 'bookstore/categories/CHECKED_STATUS';
+const INITIAL_STATE = {
+  categories: [],
+  isChecking: false,
+  message: '',
+};
+
+const checkStatus = () => ({
+  type: CHECKED_STATUS,
+});
+
+const categoryReducer = (state = INITIAL_STATE, action) => {
+  switch (action.type) {
+    case CHECKED_STATUS:
+      return {
+        ...state,
+        isChecking: true,
+        message: 'We are almost there',
```

Comment:
- [ ] Please ensure that the message sent to the user is similar to what was specified in the project requirements. Currently, you have `'We are almost there'` but it should be ``"Under construction"`` :100: